FD_frontend/src/store/PostPage/reducer.js

```diff
@@ -107,6 +108,8 @@ const postReducer = (state, action) => {
                 publicStatus: 'True'
         }
       }
+    case actions.CLEAR_STATE:
+      return []
```

Comment:
Isn't state initially a dictionary, not a list?

Reply:
For consistency, it should be changed to a dictionary `{}`, but it should have no ill side effect, since this is called only on unmount, and empty serializable data should trigger initialization of state anyway.